### ISSUE TITLE
Very slow download

### ISSUE THREAD

**Issue by ScottVordem** [ZenitH-AT/nvidia-update#15]
It takes me roughly 10 minutes to download a 645 MiB file, that should not be the case since I have really good internet with fast speed. 255 Mbps Download speed. It should not be taking that long.

I don't know if that is an issue with PowerShell itself or something is being throttled somewhere.

But wanted to make you aware.

**Comment by ZenitH-AT** [comment]
Yeah, that's strange. I will look into this when possible. I haven't had a lot of time to work on this script lately but should do soon.

I haven't had download issues on my side but I may need to rework the code.
